.github/workflows/test.yml

```
@@ -0,0 +1,26 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - main  
+      
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '11'
```

I think we should use a version between 17 and 21
```suggestion
          java-version: '17'
```